### PR TITLE
[lidl] Cleaned up spiders.

### PR DESCRIPTION
**Pull request by ckpiper**
I made lidl_at and lidl_be parent spiders as many others were the almost the exact same except for days, key, dataset_id, and dataset_name.

### CODE REVIEW

**Comment by ckpiper** [comment]
I did test all of them. And the log info was all the same. But I guess the Action refuses to test more then 15 spiders at a time.

File: locations/spiders/lidl_be.py
Comment (Cj-Malone) [inline locations/spiders/lidl_be.py:1]: I think you forgot to inherit LidlATSpider on this one?

Comment (Cj-Malone) [reply locations/spiders/lidl_be.py:1]: Oh I see, different format, but you still moved to self.days

**Comment by panasonic-mafia** [comment]
Thanks @ckpiper !

**Pull request merged**